amethyst_controls/src/components.rs

```diff
@@ -9,3 +9,19 @@ pub struct FlyControlTag;
 impl Component for FlyControlTag {
     type Storage = NullStorage<FlyControlTag>;
 }
+
+/// Add this to a camera to which you have already add the FlyControlTag to add a arc ball
```

Comment:
Rephrase

Reply:
I'd go with "To add an arc ball behaviour, add this to a camera which already has the FlyControlTag added."